modules/system/assets/js/snowboard/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@wintercms/snowboard",
+    "version": "2.0.0",
```

Comment:
Why 2.0 for the initial version?

Reply:
I guess this was left over from when the intention that this framework would replace the old one. ;)